source/Octopus.Client/IOctopusAsyncClient.cs

```diff
@@ -342,48 +340,6 @@ public interface IOctopusAsyncClient : IDisposable
         /// <returns>A fresh copy of the root document.</returns>
         Task<RootResource> RefreshRootDocument();
 
-        /// <summary>
-        /// Requests an IOctopusAsyncClient which will operate within a given space.
-        /// </summary>
-        /// <param name="spaceId">The ID of the space.</param>
-        /// <returns>An instance of IOctopusClient</returns>
-        Task<IOctopusAsyncClient> ForSpace(string spaceId);
-
-        /// <summary>
-        /// Requests an IOctopusAsyncClient which will operate within a given and the system space.
-        /// </summary>
-        /// <param name="spaceId">The ID of the space.</param>
-        /// <returns>An instance of IOctopusClient</returns>
-        Task<IOctopusAsyncClient> ForSpaceAndSystem(string spaceId);
-
-        /// <summary>
-        /// Requests an IOctopusAsyncClient which will operate within the system space only.
-        /// </summary>
-        /// <returns>An instance of IOctopusClient</returns>
-        Task<IOctopusAsyncClient> ForSystem();
-
-        /// <summary>
-        /// Determines whether the specified link exists.
-        /// </summary>
-        /// <param name="name">The name/key of the link.</param>
-        /// <returns>
-        /// <c>true</c> if the specified link is defined; otherwise, <c>false</c>.
-        /// </returns>
-        bool HasLink(string name);
-
-        /// <summary>
-        /// Gets the link with the specified name.
-        /// </summary>
-        /// <param name="name">The name/key of the link.</param>
-        /// <returns></returns>
-        /// <exception cref="System.Exception">If the link is not defined.</exception>
-        string Link(string name);
-        
-        /// <summary>
-        /// The space context of the client
-        /// </summary>
-        SpaceContext SpaceContext { get; }
-
```

Comment:
Spaces related properties will be moved, all these changes were introduced by spaces projects anyway, so nothing really change